web/test/pages/profile/profile-shared.test.tsx

```diff
@@ -141,6 +142,12 @@ describe("profile - shared", () => {
         industryId: randomHomeBasedIndustry(),
         operatingPhase: randomElementFromArray(defaultDescOperatingPhases as OperatingPhase[]).id,
       }),
+      formationData: generateFormationData({
+        formationFormData: generateFormationFormData({
+          addressMunicipality: undefined,
+          addressState: undefined,
+        }),
+      }),
```

Comment:
Spoke with @FarazA22, we're purposefully setting municipality and state to undefined to fix flaky tests.